test/test_unistd.rs

```diff
@@ -78,8 +79,8 @@ fn test_mkstemp() {
 
 #[test]
 fn test_getpid() {
-    let pid = getpid();
-    let ppid = getppid();
+    let pid: ::libc::pid_t = getpid().into();
```

Comment:
Something that we might improve later: use `NonZero<T>`, once it's stabilized (I watch relevant issue and I think it's getting close.) So the compiler would optimize `Option<Pid>` to `Pid`.

The question is whether zero PID is somehow forbidden by POSIX.